### PR TITLE
Update Guides

### PR DESCRIPTION
**Pull request by civrot**
What?
----

Adds items based on our discussion earlier today.

### CODE REVIEW

File: best-practices/README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ General
 * Don't write code that guesses at future functionality.
 * Exceptions should be exceptional.
 * [Keep the code simple].
+* Good code is part of our culture.
```
Comment (raghuvarmabh) [inline best-practices/README.md:16]: 🤔

**Review by antonow — APPROVED** [review]

**Review by mcnelson — APPROVED** [review]
Thanks for updating!

File: best-practices/README.md (hunk 1 above)
Comment (civrot) [reply best-practices/README.md:16]: Please refer to our Code Review Guidelines.

Untitled

Comment (mcnelson) [reply best-practices/README.md:16]: NO SARCASM IS NO FUN :trollface:

Comment (civrot) [reply best-practices/README.md:16]: Do you want me change it?

Comment (mcnelson) [reply best-practices/README.md:16]: Don't think so, in all seriousness, it's probably a good rule to have. At least in PR comments.

**Review by feitla — APPROVED** [review]

File: code-review/README.md [hunk 2]
```diff
@@ -22,10 +22,15 @@ Everyone
...
   solution:" comments. Post a follow-up comment summarizing offline discussion.
+* Try to make small atomic PRs to allow simpler reviews. If a "feature" requires 
+  multiple PRs at once document the dependencies and in what order they should
+  be reviewed.
+* Close stale PRs.
 
 Having Your Code Reviewed
 -------------------------
 
+* Include set up and testing instructions.
```
Comment (antonow) [inline code-review/README.md:34]: Suggestion: Include a link to the relevant PR in the case of a cross-repo dependent set of PRs.

File: code-review/README.md [hunk 3]
```diff
@@ -22,10 +22,15 @@ Everyone
   If they are, use them with aplomb.
 * Talk in person if there are too many "I didn't understand" or "Alternative
   solution:" comments. Post a follow-up comment summarizing offline discussion.
+* Try to make small atomic PRs to allow simpler reviews. If a "feature" requires 
+  multiple PRs at once document the dependencies and in what order they should
+  be reviewed.
+* Close stale PRs.
```
Comment (raghuvarmabh) [inline code-review/README.md:29]: @mcnelson @feitla @natemontgomery @antonow  should we agree on a time line to close stale pr's ?
may be like 1 month

File: code-review/README.md (hunk 2 above)
Comment (raghuvarmabh) [reply code-review/README.md:34]: 👍

**Pull request merged**